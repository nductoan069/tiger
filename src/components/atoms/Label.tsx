import {
  StyleSheet,
  Text,
} from 'react-native';

const styles = StyleSheet.create({
  label: {
    fontWeight: 'bold',
    fontSize: 32,
    fontFamily: 'Avenir',
  }
})

interface Props {
  children: string
}

function Label(props: Props) {
  return <Text style={styles.label}>{props.children}</Text>
}

export default Label;
