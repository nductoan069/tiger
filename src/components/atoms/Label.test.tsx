import React from 'react';
import renderer from 'react-test-renderer';

import Label from "./Label";

it('renders correctly', () => {
    const tree = renderer
      .create(<Label>Facebook</Label>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  
