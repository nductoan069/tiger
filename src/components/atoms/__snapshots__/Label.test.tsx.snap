// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-text-901oao r-fontFamily-9ovc40 r-fontSize-1ui5ee8 r-fontWeight-vw2c0b"
  dir="auto"
  onClick={[Function]}
>
  Facebook
</div>
`;
